docs/en/stack/data-frames/index.asciidoc

```diff
@@ -1,3 +1,4 @@
 include::dataframes.asciidoc[]
 include::ecommerce-example.asciidoc[]
-include::api-quickref.asciidoc[]
\ No newline at end of file
+include::api-quickref.asciidoc[]
+include::dataframe-limitations.asciidoc[]
```

Comment:
There's actually a "Limitations" section in the Stack Overview: https://www.elastic.co/guide/en/elastic-stack-overview/master/limitations.html
I think until we rethink the layout of this book, we should follow the existing pattern and put the "Data frames" page there too.